api-client/src/utils/getExcerpt.ts

```diff
@@ -37,27 +37,34 @@ type ExcerptOptions = {
 }
 
 const getExcerpt = (text: string, { query, maxChars, replaceLineBreaks }: ExcerptOptions): string => {
-  const match = query ? normalizeString(text).match(normalizeString(query)) : []
```

Comment:
Before it tried to also match regex strings, meaning writing `?` would break the search

Reply:
Ouch, good catch!